dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -649,15 +687,22 @@ def test_null_in_file(self):
         )
        
     def test_correct_total_sample_size_and_counts_and_mutability(self):
-        file_path = os.path.join(test_root_path, 'data', 'csv/empty_rows.txt')
-        data = pd.read_csv(file_path)
+        data = [['test1', 1.0],
```

Comment:
this makes it clear what data is being checked and ensures we are not dependent on the data in the file to exist or not change